### PR TITLE
Added Moon board game

### PR DESCRIPTION
**Pull request by parthrc**
Submitting this PR means you have:

- [x] searched the [readme][readme-url] file and searched the [open pull requests][pr-url] (no duplicates please)
- [x] and will follow [the desired format][format-url]
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines][contributing-url] or [open an issue][new-issue-url].

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

[pr-url]: https://github.com/edm00se/awesome-board-games/pulls
[readme-url]: https://github.com/edm00se/awesome-board-games/edit/main/readme.md
[format-url]: https://github.com/edm00se/awesome-board-games/blob/main/formatting.md
[contributing-url]: https://github.com/edm00se/awesome-board-games/blob/main/contributing.md
[new-issue-url]: https://github.com/edm00se/awesome-board-games/issues/new

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2202,6 +2202,18 @@ Superfight is a game where you argue with your friends over ridiculous fights. T
...
+> Over the three distinct eras of the game, players compete for majority in 5 aspects of their bases: housing, transportation, science, industry, and food production. At the end of each era, the leader in each of these areas as determined by flag icons on their constructed buildings, collects bonus victory points.
+
+![Moon](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEgDKpy-Y6LadX-TprKSQAAJ6Hu6BCDkImHEl39lCRGORuJK5-PkFSbL9UgrSaLIVbOCXncFNYBILbGXiutgMqsEIa3lAAoImvkw5BeEItOthMdNokiGaaWlswcHMocbqoeyOwQZNiDfqzUXBiRm0Pb1wW2I9P76cv2Z5a9H5fqoQB6soAL216hbdEBo/s640/Polyhedron%20Collider%20Moon%20Board%20Game%20Review.jpg)
```
Comment (edm00se) [inline readme.md:2211]: This image is too large. We target ~250px on whichever is the larger dimension, horizontal or vertical.

**Review by edm00se — CHANGES_REQUESTED** [review]
Please updated the image to one that is sized appropriately.

**Comment by parthrc** [comment]
@edm00se the closest one i could find was 246 x 300

**Review by edm00se — APPROVED** [review]
> @edm00se the closest one i could find was 246 x 300

That will work.

refs #269

**Pull request merged**